src/main/java/duke/command/List.java

```diff
@@ -0,0 +1,34 @@
+package duke.command;
+
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Encapsulates a List commands that deals with listing out all the task in the task list.
+ *
+ * @author Zhi Bin
+ * @version Duke Level 9
+ */
+public class List extends DukeCommand {
```

Comment:
Similarly ListCommand. This is where it is quite clear that this may be confusing. When creating this object we will create List list = new List() which is not very clear that this is in fact a Command rather than a list datastructure